rules/diff_test.bzl

```diff
@@ -0,0 +1,26 @@
+# Copyright 2019 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""A test rule that compares two binary files.
+
+The rule uses a Bash command (diff) on Linux/macOS/non-Windows, and a cmd.exe
+command (fc.exe) on Windows (no Bash is required).
+"""
+
+load(
+    "//rules/private:diff_test_private.bzl",
+    _diff_test = "diff_test",
+)
+
+diff_test = _diff_test
```

Comment:
Since `private/diff_test_private.bzl` has only one public symbol, both `diff_test_private.bzl` and `diff_test.bzl` are equivalent.

Maybe put all the content here and delete `diff_test_private.bzl`?

Reply:
Done.